src/generateRoutes.js

```diff
@@ -8,12 +8,28 @@ function generateRoutes(routes, app, endpointInfo = false) {
     info(`\n${route.name} endpoints`)
 
     let idFieldName = null
-    if(route.entity){
+    if (route.entity) {
```

Comment:
I believe that if I wrote tests in generateRoutes, they would be the same test cases, with the same metadata, wouldn't it be testing the same thing twice in two different places?
The tests in generateControllers generate routes with generateRoutes, that's why, even though I made big changes in generateRoutes, the code coverage didn't decrease, because the tests in generateControllers are covering 100% of the cases of generateRoutes.
If you think it makes sense, I can pass the cases from generateControllers to generateRoutes, and then make a slightly different assertion in generateControllers.